maxim/logger/llamaindex/utils.py

```
@@ -0,0 +1,101 @@
+        if hasattr(llm, "top_p"):
+            model_parameters["top_p"] = getattr(llm, "top_p", None)
+        if hasattr(llm, "frequency_penalty"):
+            model_parameters["frequency_penalty"] = getattr(llm, "frequency_penalty", None)
+        if hasattr(llm, "presence_penalty"):
+            model_parameters["presence_penalty"] = getattr(llm, "presence_penalty", None)
+        if hasattr(llm, "stop"):
+            model_parameters["stop"] = getattr(llm, "stop", None)
+
+        return model_parameters
+
+def parse_message_content(blocks: List[ContentBlock]):
```

Ruff (0.12.2)

Missing return type annotation for public function `parse_message_content`

(ANN201)